src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect2.java

```diff
@@ -14,7 +14,7 @@
 /**
  * Some Javadoc.
  * <p>
- * /^ WARN/   Some Javadoc.<p>
+ *    Some Javadoc.<p>
```

Comment:
I have accidentally removed `warn` in this PR also, originally it was part of https://github.com/checkstyle/checkstyle/pull/15717 but I guess this will not cause conflict and I was too lazy to revert it 😴